test/kuttl/common/scripts/check_debug_in_neutron_pod_logs.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+#
+found=0
+not_found=1
+if [ "$1" = "--reverse" ];then
+    # check there is no DEBUG line in the logs
+    found=1
+    not_found=0
+fi
```

Comment:
instead of changing value for $found and $not_found You can simple pass "-v" to the grep command if You are expecting to not found the DEBUG line.

Reply:
Do you mean that we can use the --reverse option to detect if we want it to be found or not?

Reply:
yes, I was thinking about this option exactly :)